workloads/templates/workload-baseline-script-cm.yml.j2

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scale-ci-workload-script
+data:
+  run.sh: |
+    #!/bin/sh
```

Comment:
I think we should add `set -e` here to all `run.sh` scripts.  @jmencak's http workload does this to aid in the event of an error within the shell script.